src/js/gi/components/search/InstitutionFilterForm.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+
+import Checkbox from '../Checkbox';
+import RadioButtons from '../RadioButtons';
+import Dropdown from '../Dropdown';
+
+export class InstitutionFilterForm extends React.Component {
+
```

Comment:
I _think_ this extra space would fail the linter; there may be a rule about not having extraneous line breaks at the beginning or end of a block. Something to look out for the other modules as well.